src/main/java/org/myrobotlab/service/IndianTts.java

```diff
@@ -36,7 +36,7 @@ public IndianTts(String n, String id) {
   @Override
   public void startService() {
     super.startService();
-    httpClient = (HttpClient) startPeer("httpClient");
+    httpClient = (HttpClient) startPeer("http");

```

Comment:
Aren't peers handled by config now?

Reply:
yup and with that comes "optional" starting .. - removing this